src/decider/agent/copyrightDeactivationClutterRemovalScript.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python3
+import spacy
+import pandas as pd
+import argparse
+import re
+import csv
+
+THE_PROBABLE_LOGIC_POS_CHECK = ['NOUN', 'NUM', 'PROPN', 'PROPN']
+THE_PROBABLE_LOGIC_NER_CHECK = ['DATE', 'PERSON', 'CARDINAL', 'ORG']
+
+def preProcessing(file, clutter_flag):
+    with open(file, 'r') as f:
+        df = pd.read_json(f, orient='records')
+        df['content'] = df['content'].str.lower()
+    copyrightDeactivationMain(df, clutter_flag)
+
+def copyrightDeactivationMain(df, clutter_flag):
+    nlp = spacy.load("en_core_web_sm")
+    ## Iterating through each row and doing preprocessing over it.
+    ## Picking out the manual tags from the csv and putting them into seperate column "Original Tag"
+    for index, row in df.iterrows():
+        text = df.loc[index, 'content']
+        doc = nlp(text)
+
+        if type(text) == float:
+            continue
+        
+        ## Lemmatization
+        lemma_list = []
+        for token in doc:
+            lemma_list.append(token.lemma_)
+        
+        # Filter the stopword
+        filtered_sentence =[] 
+        for word in lemma_list:
+            lexeme = nlp.vocab[word]
+            if lexeme.is_stop == False:
+                filtered_sentence.append(word)
+        
+        # Remove punctuation
+        punctuations="?:!.,;'"
+        for word in filtered_sentence:
+            if word in punctuations:
+                filtered_sentence.remove(word)
+
+        ## List joining and Filtering (c) and copyright unicode symbol
+        list_of_copyrights = " ".join(map(str,filtered_sentence))
+        substring = "( c )"
+        cp_symbol = '\xa9' ##Unicode for copyright Symbol
+        
+        if "copyright" not in list_of_copyrights:
+            if substring in list_of_copyrights:
+                list_of_copyrights = list_of_copyrights.replace(substring, "copyright")
+            
+            elif cp_symbol in list_of_copyrights:
```

Comment:
Please change `elif` to `if` as there can be both `( c )` and &copy; in the same line.